cdc/log.cc

```diff
@@ -389,12 +390,24 @@ bytes log_data_column_deleted_elements_name_bytes(const bytes& column_name) {
     return to_bytes(cdc_deleted_elements_column_prefix) + column_name;
 }
 
+// Makes sure that number of sstables won't grow too large
+static int calculate_window_size(int ttl) {
```

Comment:
Please document what it takes and returns. What does it take (seconds?) and what does it return (minutes?)
Or better yet (I think): just inline this code where you use it, where the context of the output as "MINUTES" is obvious, and the input is obvious, so it doesn't need to be documented.

Reply:
ok I will inline.